### PR TITLE
Local notifications

### PR DESCRIPTION
**Pull request by davidbritch**
A sample that shows how to send, receive, and display a local notification on Android, iOS, and Mac Catalyst.

### CODE REVIEW

File: 8.0/PlatformIntegration/LocalNotificationsDemo/LocalNotificationsDemo/App.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+            InitializeComponent();
+
+            MainPage = new AppShell();
+        }
+
+        //protected override Window CreateWindow(IActivationState? activationState)
+        //{
+        //    return new Window(new AppShell());
+        //    //return base.CreateWindow(activationState);
+        //}
```
Comment (jfversluis) [inline 8.0/PlatformIntegration/LocalNotificationsDemo/LocalNotificationsDemo/App.xaml.cs:16]: Should we keep this?

File: 8.0/PlatformIntegration/LocalNotificationsDemo/LocalNotificationsDemo/Platforms/MacCatalyst/Entitlements.plist [hunk 2]
```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jfversluis) [inline 8.0/PlatformIntegration/LocalNotificationsDemo/LocalNotificationsDemo/Platforms/MacCatalyst/Entitlements.plist:1]: Should we add a note somehow like with https://github.com/dotnet/maui-samples/tree/main/8.0/Apps/PointOfSale#local-testing?

**Review by jfversluis — APPROVED** [review]

File: 8.0/PlatformIntegration/LocalNotificationsDemo/LocalNotificationsDemo/App.xaml.cs (hunk 1 above)
Comment (davidbritch) [reply 8.0/PlatformIntegration/LocalNotificationsDemo/LocalNotificationsDemo/App.xaml.cs:16]: That's a reminder that I still need to tinker with the sample on Android.

Comment (davidbritch) [reply 8.0/PlatformIntegration/LocalNotificationsDemo/LocalNotificationsDemo/App.xaml.cs]: Removed.

File: 8.0/PlatformIntegration/LocalNotificationsDemo/LocalNotificationsDemo/Platforms/MacCatalyst/Entitlements.plist (hunk 2 above)
Comment (jfversluis) [reply 8.0/PlatformIntegration/LocalNotificationsDemo/LocalNotificationsDemo/Platforms/MacCatalyst/Entitlements.plist:1]: Looks like this builds for this project, so let's ignore for now. Can't remember why we needed to do that for the other project(s).

**Pull request merged**